core/server/data/migrations/versions/3.7/01-fix-incorrect-member-labels-foreign-keys.js

```diff
@@ -0,0 +1,54 @@
+const logging = require('../../../../lib/common/logging');
+const commands = require('../../../schema').commands;
+const config = require('../../../../config');
+
+const table = 'members_labels';
+const message1 = 'Adding table: ' + table;
+const message2 = 'Dropping table: ' + table;
+
+// 3.6.0 had an incorrect schema definition that created foreign key constraints for the wrong table.
+//
+// The schema.js is correct as of 3.7.0 and members_labels has not been used at this point
+// so it's safe to drop and recreate the table to let knex do it's thing.
+
+const dropTable = function (connection) {
+    return connection.schema.hasTable(table)
+        .then(function (exists) {
+            if (!exists) {
+                logging.warn(message2);
+                return;
+            }
+
+            logging.info(message2);
+            return commands.deleteTable(table, connection);
+        });
+};
+
+const addTable = function (connection) {
+    return connection.schema.hasTable(table)
+        .then(function (exists) {
+            if (exists) {
+                logging.warn(message1);
+                return;
+            }
+
+            logging.info(message1);
+            return commands.createTable(table, connection);
+        });
+};
+
+module.exports.up = ({connection}) => {
+    // SQLite did not suffer from the bug meaning the members_labels table may
```

Comment:
Not sure if this is safe to assume. The table created in SQLite has following definition on 3.6.0:
```
CREATE TABLE `members_labels` (`id` varchar(24) not null, `member_id` varchar(24) not null, `label_id` varchar(24) not null, `sort_order` integer not null default '0', foreign key(`member_id`) references `posts`(`id`), foreign key(`label_id`) references `tags`(`id`), primary key (`id`));
```
Though the functionality work just fine with this structure :man_shrugging:

Reply:
I've updated to drop+recreate the tables on SQLite too as per discussion on Slack.